tests/docker/php/Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG PHP_VERSION
+FROM php:${PHP_VERSION}-cli-alpine
+
+RUN apk --no-cache add \
+    git \
+    bash \
+    linux-headers \
+    > /dev/null
+
+# install xDebug
+ENV XDEBUG_MODE=coverage
+ENV XDEBUG_CONFIG=""
+RUN apk add --update --no-cache --virtual .build-dependencies $PHPIZE_DEPS \
+    && docker-php-ext-install pcntl sockets \
+    && pecl install xdebug \
+    && docker-php-ext-enable xdebug \
+    && pecl clear-cache \
+    && apk del .build-dependencies
+
+RUN curl -sS https://getcomposer.org/installer | php -- --2 --install-dir=/usr/local/bin --filename=composer
```

Comment:
I suggest replacing it with this
```
COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
```

Reply:
Why is it better?

Reply:
It's advice from composer docs: https://getcomposer.org/doc/00-intro.md#docker-image

Reply:
I agree. Using installer may lead to unexpected downgrade of PHP version. I've experienced that.